src/util/util.ts

```diff
@@ -27,7 +27,12 @@ export function currentTimeStamp(): string {
 	return moment().format("YYYY-MM-DDTHH:mm:ss.SSSZZ");
 }
 
-export const numberOfOsProcesses = os.cpus().length;
+export const numberOfOsProcesses = (() => {

```

Comment:
The reason I have done it this way is that newer versions of TypeScript/node typings want don't like this on a potentially undefined variable. For now it is okay and I have changed it to a ternary expression as requested.